src/server/game/map/tile/city.ts

```diff
@@ -16,7 +16,7 @@ export class City {
     this.civID = civID;
 
     this.tiles = new Set();
-    this.tiles.add(center);
+    this.addTile(center);
```

Comment:
Is there a reason to make this a function?

Reply:
The function already existed for other reasons, it just bothered me that the function existed but wasn't used here.